### PR TITLE
Move SCD30 into air quality

### PR DESCRIPTION
**Pull request by sjtrny**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [X] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [X] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Review by jesserockz — CHANGES_REQUESTED** [review]
No reason it cannot be in both =)

**Comment by sjtrny** [comment]
> No reason it cannot be in both =)

Done!

File: index.rst [hunk 1]
```diff
@@ -268,8 +269,8 @@ Environmental
     MS5611, components/sensor/ms5611, ms5611.jpg, Pressure
     NTC Thermistor, components/sensor/ntc, ntc.jpg, Temperature
     RuuviTag, components/sensor/ruuvitag, ruuvitag.jpg, Temperature & Humidity & Accelerometer
-    SCD30, components/sensor/scd30, scd30.jpg, Temperature & Humidity
     SDP3x, components/sensor/sdp3x, sdp31.jpg, Pressure
+    SCD30, components/sensor/scd30, scd30.jpg, CO2 & Temperature & Humidity
```
Comment (oxan) [inline index.rst:272]: Can we move it up again to retain the alphabetical sorting?

**Review by oxan — CHANGES_REQUESTED** [review]

File: index.rst (hunk 1 above)
Comment (sjtrny) [reply index.rst:272]: Done!

**Review by jesserockz — APPROVED** [review]

**Review by oxan — APPROVED** [review]

**Pull request merged**